pds_github_util/issues/RstRddReport.py

```diff
@@ -160,7 +160,7 @@ def create(self, repos):
     def _non_bug_metrics(self, type, repo):
         for issue in repo.issues(
                 state='closed',
-                labels=type,
+                labels=f'{self._build},{type}',
```

Comment:
Not sure what this has to do with issue moving, but it looks good to me 👍

Reply:
Right @nutjob4life , this was a bug correction I missed to push earlier